ngraph/core/src/op/floor.cpp

```diff
@@ -81,3 +79,5 @@ bool op::Floor::evaluate(const HostTensorVector& outputs, const HostTensorVector
     NGRAPH_OP_SCOPE(v0_Floor_evaluate);
     return floorop::evaluate_floor(inputs[0], outputs[0], shape_size(get_output_shape(0)));
 }
+
+NGRAPH_RTTI_DEFINITION(op::v0::Floor, "Floor", 0);
```

Comment:
```suggestion
NGRAPH_RTTI_DEFINITION(op::v0::Floor, "Floor", 0, util::UnaryElementwiseArithmetic);
```
AFAIK we should add also parent class in case of such inheritance. @ilyachur could you confirm?

Reply:
Yes @mitruska you are right!

Reply:
Updated:)

Reply:
Great, thanks:) Last thing, just for consistency please move this line at the begining of the file as it is done for other ops (right after usings, in place of the old const expression)